arwen/contexts/storage.go

```diff
@@ -3,16 +3,20 @@ package contexts
 import (
 	"bytes"
 	"errors"
+	"fmt"
 
 	"github.com/ElrondNetwork/elrond-go-core/core/check"
 	logger "github.com/ElrondNetwork/elrond-go-logger"
 	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
 	"github.com/ElrondNetwork/wasm-vm-v1_4/arwen"
+	"github.com/ElrondNetwork/wasm-vm-v1_4/config"
 	"github.com/ElrondNetwork/wasm-vm-v1_4/math"
 )
 
 var logStorage = logger.GetOrCreate("arwen/storage")
 
+const minimumStorageLoadCost = 10000
```

Comment:
Shouldn't this depend on the `GasSchedule`?  It should be decided at runtime, not as a constant, but rather as one of either:
* the base gas cost of a `StorageLoad`, without adding the cost per byte (as defined in the `GasSchedule`)
* the base gas cost of a `StorageLoad`, but also adding the cost per byte

Either possibility would be good. But the constant 10000 seems arbitrary.

Reply:
Added minimumStorageLoadCost to the gasSchedule